packages/salesforcedx-vscode-lwc/src/commands/forceLightningLwcCreate.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import {
+  CancelResponse,
+  ContinueResponse,
+  DirFileNameSelection,
+  PostconditionChecker
+} from '@salesforce/salesforcedx-utils-vscode/out/src/types';
+import * as path from 'path';
+import { Observable } from 'rxjs/Observable';
+import * as vscode from 'vscode';
+import { nls } from '../messages';
+
+const sfdxCoreExports = vscode.extensions.getExtension(
```

Comment:
Here is how we import it.
The downside of this is that all of these things are exported as `any` so we lose the type safety... 😞